### ISSUE TITLE
Chintaman Navsha Wanaga

### ISSUE THREAD

**Issue by sohelrodrigues** [netneutrality/savetheinternet.in#141]
Both the given email addresses aren't reachable

1> n.wanaga@sansad.nic.in
2> chintamanvanaga@yahoo.in

**Comment by miteshashar** [comment]
@sohelrodrigues 
1. You've mentioned the first email address as `n.wanaga@sansad.nic.in`. It was supposed to be `cn.wanaga@sansad.nic.in`.
2. What is the reason of failure cited in the Delivery Failure Notification email?

**Comment by sohelrodrigues** [comment]
Hi,
I tried sending the mail to the following addresses, and received a mailer daemon message. I thought that maybe the yahoo domain maybe incorrectly typed as"yahoo.in" instead of "yahoo.co.in" but it didn't help. Also I tried using both spellings of the surname- "Wanaga" and "Vanaga" but the issue was unresolved. Below are the failure messages I received.

Delivery to the following recipient failed permanently:
cn.vanaga@sansad.nic.in
Technical details of permanent failure:
Google tried to deliver your message, but it was rejected by the server for the recipient domain sansad.nic.in by mailgw.nic.in. [164.100.2.6].
The error that the other server returned was:
550 #5.1.0 Address rejected.
-------------------------------------------------------------------------------------------------------------------------------
Delivery to the following recipient failed permanently:
cn.wanaga@sansad.nic.in
Technical details of permanent failure:
Google tried to deliver your message, but it was rejected by the server for the recipient domain sansad.nic.in by mailgw.nic.in. [164.100.2.6].
The error that the other server returned was:
550 #5.1.0 Address rejected.
-------------------------------------------------------------------------------------------------------------------------------
Delivery to the following recipient failed permanently:
chintamanvanaga@yahoo.in
Technical details of permanent failure:
Google tried to deliver your message, but it was rejected by the server for the recipient domain yahoo.in by mx-apac.mail.gm0.yahoodns.net. [106.10.166.52].
The error that the other server returned was:
554 delivery error: dd This user doesn't have a yahoo.in account (chintamanvanaga@yahoo.in) [0] - mta1021.mail.sg3.yahoo.com
-------------------------------------------------------------------------------------------------------------------------------
Message from yahoo.co.in.
Unable to deliver message to the following address(es).
<chintamanvanaga@yahoo.co.in>:
This user doesn't have a yahoo.co.in account (chintamanvanaga@yahoo.co.in) [0]
-------------------------------------------------------------------------------------------------------------------------------
Delivery to the following recipient failed permanently:
cn.wanaga@sansad.nic.in
Technical details of permanent failure:
Google tried to deliver your message, but it was rejected by the server for the recipient domain sansad.nic.in by mailgw.nic.in. [164.100.10.19].
The error that the other server returned was:
550 #5.1.0 Address rejected.
-------------------------------------------------------------------------------------------------------------------------------
Delivery to the following recipient failed permanently:
chintamanvanaga@yahoo.in
Technical details of permanent failure:
Google tried to deliver your message, but it was rejected by the server for the recipient domain yahoo.in by mx-apac.mail.gm0.yahoodns.net. [106.10.166.52].
The error that the other server returned was:
554 delivery error: dd This user doesn't have a yahoo.in account (chintamanvanaga@yahoo.in) [0] - mta1016.mail.sg3.yahoo.com
-------------------------------------------------------------------------------------------------------------------------------

Thanks in advance.

Sincerely,
Sohel Rodrigues.
